shotover-proxy/src/transforms/cassandra/sink_single.rs

```diff
@@ -44,6 +44,7 @@ pub struct CassandraSinkSingle {
     chain_name: String,
     failed_requests: Counter,
     tls: Option<TlsConnector>,
+    pushed_messages_tx: Option<tokio::sync::mpsc::UnboundedSender<Messages>>,
```

Comment:
Does this get used anywhere?

Reply:
In the constructor of `CassandraConnection`.

Reply:
Can this be resolved?